packages/web-components/src/components/ic-text-field/ic-text-field.tsx

```diff
@@ -354,25 +354,19 @@ export class TextField {
 
   private hasLeftIconSlot(): boolean {
     const iconEl = this.el.querySelector(`[slot="icon"]`);
-    if (iconEl != null) {
-      return true;
-    }
-    return false;
+    return iconEl !== null ? true : false;
```

Comment:
Don't think `? true : false` is required as that should be returned by `iconEl !== null`